views/skala3ma-layout.html

```diff
@@ -102,6 +102,8 @@
 
     <!-- script for custom color picker-->
     <script src="/public/js/huebee.pkgd.min.js"></script>
```

Comment:
did you completely remove huebee? or is this line still required?

Reply:
done in latest commit